### PR TITLE
feat: set k8s tags for sentry events

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: https://github.com/lightdash/lightdash/issues/10294

### Description:

Sets tags for: 
```
  podName
  podNamespace
  nodeName
  lightdashCloudInstance
```

from envs: 

```
LIGHTDASH_CLOUD_INSTANCE
K8S_NODE_NAME
K8S_POD_NAME
K8S_POD_NAMESPACE

```

Triggered an error and saw the tags being passed (when they're not `undefined`): 

Screenshot 2024-06-03 at 16 45 17

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/App.ts [hunk 1]
```diff
@@ -535,6 +535,14 @@ export default class App {
             }) as RequestHandler,
         );
         expressApp.use(Sentry.Handlers.tracingHandler());
+
+        // Set k8s tags for Sentry
+        Sentry.setTags({
+            podName: this.lightdashConfig.k8s.podName,
```
Comment (owlas) [inline packages/backend/src/App.ts:541]: can we make tags camel case, they're easier to read in the sentry UI imo. Also we should mention these are from kubernetes: `k8s_pod_name`

**Review by owlas — APPROVED** [review]
approve but suggest naming for sentry UI

File: packages/backend/src/App.ts (hunk 1 above)
Comment (joaoviana) [reply packages/backend/src/App.ts:541]: sounds good!

Comment (joaoviana) [reply packages/backend/src/App.ts]: renamed here: https://github.com/lightdash/lightdash/pull/10299/commits/2431a320f626133d2f764ff8dd715f8771853015

**Review by owlas — APPROVED** [review]

**Pull request merged**